ui/publicTags/submit/result/PublicTagsSubmitResultSuccess.tsx

```diff
@@ -0,0 +1,53 @@
+import { Box, Flex, Grid, GridItem } from '@chakra-ui/react';
+import React from 'react';
+
+import type { FormSubmitResultGrouped } from '../types';
+
+import AddressEntity from 'ui/shared/entities/address/AddressEntity';
+import EntityTag from 'ui/shared/EntityTags/EntityTag';
+
+interface Props {
+  data: FormSubmitResultGrouped;
+}
+
+const PublicTagsSubmitResultSuccess = ({ data }: Props) => {
+  return (
+    <Grid gridTemplateColumns={{ base: '1fr', lg: '1fr 1fr' }} rowGap={ 3 } columnGap={ 3 }>
+      <GridItem overflow="hidden">
+        <Box fontSize="sm" color="text_secondary" fontWeight={ 500 }>Smart contract / Address (0x...)</Box>
```

Comment:
maybe the text should be "contracts/addresses" and "tags" (or depend on the addresses and tags length)

Reply:
In the design, it is always singular. It seems to work as a table header even though it doesn't look like it.